Mathlib/Topology/ContinuousMap/StoneWeierstrass.lean

```diff
@@ -310,6 +310,43 @@ theorem exists_mem_subalgebra_near_continuous_of_separatesPoints (A : Subalgebra
   use g
   rwa [norm_lt_iff _ pos] at b
 
+-- Redefine `X`, since for the next theorem it need not be compact
+variable {X : Type*} [TopologicalSpace X]
```

Comment:
Since you use it in one theorem only, you could put that type in the theorem declaration instead of a `variable` line.